images/manila-provisioner/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openshift/origin-release:golang-1.10
+COPY . /go/src/k8s.io/cloud-provider-openstack
+RUN cd /go/src/k8s.io/cloud-provider-openstack && \
+    go build -o manila-provisioner cmd/manila-provisioner/main.go
+
+FROM centos:7
```

Comment:
how about registry.svc.ci.openshift.org/openshift/origin-v4.0:base ?

Reply:
You're right. I will update both the Dockerfiles.